html/rendering/widgets/baseline-alignment-and-overflow.tentative.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<title>HTML: widgets' baseline alignment and interaction with 'overflow'</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+.test {
+  border-collapse: collapse;
+  font-size: 10px;
+}
+.test td {
+  border: none;
+  padding: 0;
+  margin:0;
+  outline: 1px solid silver;
+}
+
+.test td > input,
+.ref td > i,
+.ref td > b,
+.ref td > img,
+.ref td > button {
+  font: inherit;
+  height: 60px;
+  width: 60px;
+  padding: 10px;
+  box-sizing: border-box;
+  margin: 10px 0;
+}
+.ref button img {
+  height: 100%;
+  width: 100%;
+  display: block;
+}
+.ref i {
+  display: inline-grid;
+  border: 2px solid;
+  align-items: center;
+}
+.ref b {
+  display: inline-block;
+}
+.ref td > img.no-margin-bottom {
+  margin-bottom: 0;
+}
+[style*="appearance: none;"] {
+  -webkit-appearance: none; /* TODO(zcorpan) remove this when unprefixed appearance is supported */
+}
+</style>
+<div id="log"></div>
+<h2>refs</h2>
+<table class="test ref">
+  <tr class="ref-text"><td><span>ref</span> <i>text</i>
```

Comment:
I find these names fairly confusing. Why is text the name of the ref for the i tag? Why is checkbox the name of the ref for the img.no-margin-bottom tag? What concepts are these really representing? Is the idea that images with alts should somehow match `<b>` tags, and thus you're using that as a standin? That definitely needs a comment, if so.